### PR TITLE
[Botkit] Ignore CS1998 warning in not implemented methods.

### PR DESCRIPTION
**Pull request by ceciliaavila**
## Proposed Changes
- Add **CS1998 warning** disable statements
- Add throw _NotImplementedException_ in not implemented method

### CODE REVIEW

**Comment by denscollo** [comment]
Could be that the indentation is wrong?

**Comment by GasparAcevedoZainSouthworks** [comment]
> Could be that the indentation is wrong?

Hi @denscollo, AFAIK, the pre-processor directives are always left-aligned. Check this [link](https://github.com/OmniSharp/omnisharp-vscode/issues/2008#issuecomment-363153560) for more info.

**Comment by denscollo** [comment]
> 
> 
> > Could be that the indentation is wrong?
> 
> Hi @denscollo, AFAIK, the pre-processor directives are always left-aligned. Check this [link](https://github.com/OmniSharp/omnisharp-vscode/issues/2008#issuecomment-363153560) for more info.

Great, thank you for the info! I thought that it'd something like that, but I wanted to be sure.

File: libraries/Microsoft.BotKit/BotkitDialogWrapper.cs [hunk 1]
```diff
@@ -24,21 +25,26 @@ public BotkitDialogWrapper(DialogContext dialogContext, IBotkitConversationStep
         /// <summary>
         /// Gets or sets an object containing variables and user responses from this conversation.
         /// </summary>
+        /// <value> An object with variables. </value>
```
Comment (GasparAcevedoZainSouthworks) [inline libraries/Microsoft.BotKit/BotkitDialogWrapper.cs]: I think that the whitespaces at the beginning and the End of the description should be removed, just for having consistency with the other tags. Should be something like this:
```
<value>Description.</value>
```

**Pull request merged**